service/src/test/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderServiceTest.java

```diff
@@ -300,18 +305,28 @@ void shouldRemoveOrderPropertiesWhenTheyExistInCaseDetails() {
                 }
                 break;
             case SUPERVISION_ORDER:
-                final String suffix = dateFormatterService.getDayOfMonthSuffix(date.getDayOfMonth());
-                final String formattedDateTime =
-                    dateFormatterService.formatLocalDateTimeBaseUsingFormat(date.plusMonths(5),
-                        "h:mma 'on the' d'" + suffix + "' MMMM y");
                 expectedMap
-                    .put("orderType", SUPERVISION_ORDER)
-                    .put("orderTitle", "Supervision order")
-                    .put("childrenAct", "Section 31 and Paragraphs 1 and 2 Schedule 3 Children Act 1989")
-                    .put("orderDetails",
-                        String.format(
-                            "It is ordered that Example Local Authority supervises the child for 5 months from the "
-                                + "date of this order until %s.", formattedDateTime));
+                    .put("orderType", SUPERVISION_ORDER);
+                if (subtype == INTERIM) {
+                    expectedMap
+                        .put("orderTitle", "Interim supervision order")
+                        .put("childrenAct", "Section 38 and Paragraphs 1 and 2 Schedule 3 Children Act 1989")
+                        .put("orderDetails",
+                            "It is ordered that Example Local Authority supervises the child until the end of the "
+                                + "proceedings");
+                } else if (subtype == FINAL) {
```

Comment:
Same comment about `if` here

Reply:
We decided not to change it, better to be safe